packages/dev/core/src/ShadersWGSL/iblShadowsCdfx.fragment.fx

```diff
@@ -0,0 +1,17 @@
+#define PI 3.1415927
+varying vUV: vec2f;
+
+var cdfy: texture_2d<f32>;
+
+@fragment
+fn main(input: FragmentInputs) -> FragmentOutputs {
+
+    var cdfyRes = textureDimensions(cdfy, 0);
+    var currentPixel = vec2u(input.position.xy);
```

Comment:
shader preprocessor can't handle this expression see [here](https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Engines/WebGPU/webgpuShaderProcessorsWGSL.ts#L280)
```suggestion
    var currentPixel = vec2u(fragmentInputs.position.xy);
```